react-native-purchases-ui/src/index.tsx

```diff
@@ -11,6 +11,13 @@ const LINKING_ERROR =
 
 const RNPaywalls = NativeModules.RNPaywalls;
 
+export interface PresentPaywallIfNeededParams {
```

Comment:
Hmm do we need to do the same for the `presentPaywall` method? Since that would also accept some parameters which might coincide with these?

Reply:
Also, should we add this type to the UI tests?

Reply:
The presentPaywall will have some but not all of the options, for example `offering` will be in both, but not `requiredEntitlementIdentifier`

Maybe we can do something for the common ones?